### ISSUE TITLE
hello, i have a question

### ISSUE THREAD

**Issue by yeong-dev** [yuxiangsun/RTFNet#9]

**Comment by yeong-dev** [comment]
i trained with your code and didnt touch any codes but the performance was much worse than your result in paper. 
The best recall value was 0.55 and best iou value was 0.45.
i wonder if there is any other methods to be used in training.
Especially the class 'bump' recall and iou value is super lower (about 0.005)
plus, how many epochs did you train with your code? 

Thanks for this nice paper :)

**Comment by yuxiangsun** [comment]
You may want to train again.
I train until the val loss converges.

**Comment by yeong-dev** [comment]
i appreciate your kind answer.
when i checked tensorboard, val loss oscillated but train loss converged. I thought the problem was overfitting to training dataset. (it was about 300 epochs)
maybe i waited more time. Thanks.
By the way, README says that the mf dataset used in this paper was preprocessed. Can i ask what becomes different from original MFnet dataset?

**Comment by yuxiangsun** [comment]
mainly the flip operation

**Comment by SunnyWuYang** [comment]
@yeong5366 Hello, I encountered the same problem as you, I trained ResNet50(pertained) more than once, with the same configuration as the paper (batch size=3, start lr=0.01), but every time the performance is lower than the paper ( mAcc is around 0.54, mIoU is around 0.44), did you finally achieve the performance described in the paper? If it does, 
what configuration are you using? And how many times did you train in the end?  Thanks!
